### PR TITLE
[zh-CN] sync object-position with en-US

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

sync object-position translation with en-US document

### Motivation

sync and update version

### Additional details

None

### Related issues and pull requests

### CODE REVIEW

File: files/zh-cn/web/css/object-position/index.md [hunk 1]
```diff
@@ -31,26 +49,30 @@ object-position: unset;
 
 > **备注：** 这些定位方式允许被替换元素的对象被定位到内容框外部。
 
-### 形式语法
+## 正式定义
```
Comment (fwqaaq) [inline files/zh-cn/web/css/object-position/index.md:52]:
```suggestion
## 形式定义
```

File: files/zh-cn/web/css/object-position/index.md [hunk 2]
```diff
@@ -73,16 +95,14 @@ img {
 
 第一个图像的左边缘距元素框左边缘 10 个像素。第二个图像的右边缘与元素框的右边缘齐平，并位于元素框高度下方 10％处。
```
Comment (fwqaaq) [inline files/zh-cn/web/css/object-position/index.md:96]:
```suggestion
第一个图像的左边缘距元素框左边缘 10 个像素。第二个图像的右边缘与元素框的右边缘齐平，并位于元素框高度下方 10％ 处。
```

File: files/zh-cn/web/css/object-position/index.md [hunk 3]
```diff
@@ -5,22 +5,40 @@ slug: Web/CSS/object-position
...
 
 {{EmbedInteractiveExample("pages/css/object-position.html")}}
 
 ## 语法
 
 ```css
-/* <position> values */
-object-position: center top;
-object-position: 100px 50px;
+/* Keyword values */
```
Comment (yin1999) [inline files/zh-cn/web/css/object-position/index.md:17]:
```suggestion
/* 关键字值 */
```

File: files/zh-cn/web/css/object-position/index.md [hunk 4]
```diff
@@ -5,22 +5,40 @@ slug: Web/CSS/object-position
...
+object-position: top;
+object-position: bottom;
+object-position: left;
+object-position: right;
+object-position: center;
+
+/* <percentage> values */
```
Comment (yin1999) [inline files/zh-cn/web/css/object-position/index.md:24]: 下同，请翻译一下

```suggestion
/* <percentage> 值 */
```

File: files/zh-cn/web/css/object-position/index.md [hunk 5]
```diff
@@ -71,18 +93,16 @@ img {
 }
 ```
 
-第一个图像的左边缘距元素框左边缘 10 个像素。第二个图像的右边缘与元素框的右边缘齐平，并位于元素框高度下方 10％处。
+第一个图像的左边缘距元素框左边缘 10 个像素。第二个图像的右边缘与元素框的右边缘齐平，并位于元素框高度下方 10％ 处。
 
-### 结果
+#### 结果
 
 {{ EmbedLiveSample('示例', '100%','300px') }}
```
Comment (yin1999) [inline files/zh-cn/web/css/object-position/index.md:100]:
```suggestion
{{ EmbedLiveSample('定位图像内容', '100%','300px') }}
```

File: files/zh-cn/web/css/object-position/index.md [hunk 6]
```diff
@@ -71,18 +93,16 @@ img {
 }
 ```
 
-第一个图像的左边缘距元素框左边缘 10 个像素。第二个图像的右边缘与元素框的右边缘齐平，并位于元素框高度下方 10％处。
+第一个图像的左边缘距元素框左边缘 10 个像素。第二个图像的右边缘与元素框的右边缘齐平，并位于元素框高度下方 10％ 处。
```
Comment (yin1999) [inline files/zh-cn/web/css/object-position/index.md:96]:
```suggestion
第一个图像的左边缘距元素框左边缘 10 个像素。第二个图像的右边缘与元素框的右边缘齐平，并位于元素框高度下方 10% 处。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**